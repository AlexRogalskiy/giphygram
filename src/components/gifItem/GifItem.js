import React from 'react';

export class GifItem extends React.Component {
  render() {
    return (
      <div>
        Gif Item
      </div>
    );
  }
}
