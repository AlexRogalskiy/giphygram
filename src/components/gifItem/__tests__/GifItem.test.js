import React from 'react';
import renderer from 'react-test-renderer';
import { GifItem } from '../GifItem';
import searchResultsMock from '../../../mocks/searchResults';

describe('GifItem', () => {
  it('should be rendered correctly', () => {
    const gif = searchResultsMock.data[0];
    const tree = renderer
      .create(<GifItem gif={gif}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
