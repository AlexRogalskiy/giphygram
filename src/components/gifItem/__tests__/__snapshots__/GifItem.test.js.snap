// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifItem should be rendered correctly 1`] = `
<div
  className="card mb-3"
>
  <a
    href="https://giphy.com/gifs/k-dancing-3o6UBlHJQT19wSgJQk"
  >
    <img
      alt="k GIF"
      border="0"
      className="card-img-top"
      src="https://media0.giphy.com/media/3o6UBlHJQT19wSgJQk/200w.gif"
      title="k GIF"
    />
  </a>
</div>
`;
